src/parser/jobs/rdm/modules/Procs.tsx

```diff
@@ -136,6 +154,42 @@ export class Procs extends CoreProcs {
 		</Trans>
 	}
 
+	private getCriticalProcContent(id: number) {
+		if (id === this.data.statuses.GRAND_IMPACT_READY.id) {
+			return <Trans id="rdm.procs.suggestions.grandimpact.content">
+				Try to consume <DataLink status="GRAND_IMPACT_READY"/> before it expires as <DataLink action="GRAND_IMPACT"/> Gives 3 White & Black Mana and is one of your strongest GCDs outside of your Finisher Combo.
+			</Trans>
+		}
+		if (id === this.data.statuses.PREFULGENCE_READY.id) {
+			return <Trans id="rdm.procs.suggestions.prefulgence.content">
+				Try to consume <DataLink status="PREFULGENCE_READY"/> before it expires as <DataLink action="PREFULGENCE"/> is your strongest skill.
+			</Trans>
+		}
+		if (id === this.data.statuses.THORNED_FLOURISH.id) {
+			return <Trans id="rdm.procs.suggestions.viceofthorns.content">
+				Try to consume <DataLink status="THORNED_FLOURISH"/> before it expires as <DataLink action="VICE_OF_THORNS"/> Gives 3 White & Black Mana and is one of your strongest GCDs outside of your Finisher Combo.
+			</Trans>
+		}
+	}
```

Comment:
There's no real value in this separation here, since it's just hard coded Trans tags.  It'd be clearer to just put the respective Trans tags directly in their suggestion.

Reply:
Moved and Removed.